### PR TITLE
add release process

### LINKED ISSUE #671: Define release process
Things are still progressing fairly rapidly.

Define a release process for tagging this and the exposure-notifications-verification-server repositories on the same release cadence.

/assign

### PR DESCRIPTION
**Pull request by mikehelmick**
Fixes #671 

## Proposed Changes

* write down release process that was used to cut v0.2.0

**Release Note**

```release-note
Formalized release process.
```

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Exposure Notifications Servers Release Process
+
+  * Releases are cut every Tuesday (target time 9am, US Pacific Time)
+  * All releases will have a release branch and a corresponding tag
+  * Both the exposure-notifications-sever and exposure-notifications-verification-server wil follow the same release numbering
+
+# Dependenceis
```
Comment (chaodaiG) [inline RELEASING.md:7]:
```suggestion
# Dependencies
```

File: RELEASING.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Exposure Notifications Servers Release Process
+
+  * Releases are cut every Tuesday (target time 9am, US Pacific Time)
+  * All releases will have a release branch and a corresponding tag
+  * Both the exposure-notifications-sever and exposure-notifications-verification-server wil follow the same release numbering
```
Comment (icco) [inline RELEASING.md:5]: nit: s/wil/will

File: RELEASING.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+```
+
+## Update the release notes
+
+Edit the generated relnotes file and add anything that may have been missed. It's also wise to call out
+database migrations, possible breaking changes, and new environment variables that are are needed.
+
+## Create a release branch
+
+Ensure that you are on main, that you are up to date (the commit you want to cut the release at). This requires repo admin privelages.
```
Comment (icco) [inline RELEASING.md:32]: nit s/privelages/privileges/

**Comment by icco** [comment]
Note for readers, see also https://github.com/google/exposure-notifications-verification-server/pull/184

**Comment by mikehelmick** [comment]
> Note for readers, see also [google/exposure-notifications-verification-server#184](https://github.com/google/exposure-notifications-verification-server/pull/184)

pulled verification release process into this doc

**Pull request merged**